selenium_webdriver/src/main/java/core/enums/DriverType.java

```diff
@@ -0,0 +1,26 @@
+package core.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.apache.commons.lang3.NotImplementedException;
+
+import java.util.Arrays;
+
+@AllArgsConstructor
+@Getter
+public enum DriverType {
+    CHROME("chrome"),
+    FIREFOX("firefox"),
+    EDGE("edge"),
+    CHROME_REMOTE("chrome_remote");
+
+    private String browser;
+
+    public static DriverType getType(String browser) {
+
+        return Arrays.stream(DriverType.values())
+                .filter(type -> type.getBrowser().equals(browser))
+                .findFirst()
+                .orElseThrow(() -> new NotImplementedException());
```

Comment:
here you may add "No supported Webdriver were found. .... in Exeption Contractor

Reply:
Fixed